site/docs/4.6.0/api/ledger-api.md

```diff
@@ -264,6 +264,88 @@ Result: 2
 # etc
 ```
 
+## New API
+
+Since 4.6 BookKeeper provides a new client API which leverages Java CompletableFuture facility.
```

Comment:
you need also document 

- delete op
- append api in WriteHandle
- read api in ReadHandle

Also you need to update ledgeradv api page as well.